onnx2kerastl/converter.py

```diff
@@ -209,7 +209,10 @@ def onnx_to_keras(onnx_model, input_names, name_policy=None, verbose=True, chang
             else:
                 keras_names = keras_names[0]
                 node_names.append(keras_names)
-
+            cleaned_node_name = node.name.replace("#", "_").rstrip("/").lstrip("/").replace(":", "_")
```

Comment:
Maybe we can create some list with characters to remove and use regular expressions for that.
will be easier to maintain